discovery-infra/test_infra/consts.py

```diff
@@ -35,6 +35,7 @@
 PODMAN_FLAGS = "--cgroup-manager=cgroupfs --storage-driver=vfs --events-backend=file"
 LOG_FOLDER = "/tmp/assisted_test_infra_logs"
 DEFAULT_OPENSHIFT_VERSION = "4.7"
+DEFAULT_OPENSHIFT_INSTALL_RELEASE_IMAGE = "quay.io/openshift-release-dev/ocp-release:4.7.5-x86_64"
```

Comment:
I would prefer not to maintain another OCP version data. This default should be taken from the defaults JSON in assisted-service
 https://github.com/openshift/assisted-service/blob/4a57d53684600660d7d577e2e74b2d7e112503fe/default_ocp_versions.json#L9-L16